src/mudlet.cpp

```diff
@@ -3297,6 +3297,9 @@ void mudlet::createMapper(bool loadDefaultMap)
     auto hostName(pHost->getName());
     pHost->mpDockableMapWidget = new QDockWidget(tr("Map - %1").arg(hostName));
     pHost->mpDockableMapWidget->setObjectName(QStringLiteral("dockMap_%1").arg(hostName));
+    // Values need to be copies across so they are in place when the 2D
+    // mapper is created:
+    pHost->mpMap->initialiseMembersFromHost();
```

Comment:
I don't think we should change the design of how we read from the Host to this. It seems like more work and more error-prone. Just read when you need to and that's it...

Reply:
I will have to check but I *think* I may use the values in the `TMap` in the constructor for the `T2DMap` instance (which is created as part of the construction of the `dlgMapper` class)...

... actually I realise I may not have done the necessary in the other place where that construction happens - for the Mapper embedded in the main `TConsole` - :worried:

Reply:
Right, see - this way of setting values is error-prone. Let's not design like this, let's have stick to the simpler and more robust design we already have (read from the host when you need to, cache at the start of the method if that's necessary).